release-team/role-handbooks/enhancements/README.md

```diff
@@ -131,60 +131,58 @@ It is important that this process be followed and documentation remain up-to-dat
-
-  - For non-participating SIGs, mention the process and deadline for adding Enhancements to the `KEP Collection` tab on the tracking spreadsheet.  An example outreach on Slack could be structured as follows:
-
-    > Hello sig-{{ SIG_NAME }}. Enhancements team here.  Just checking in as we approach enhancements freeze at {{ FREEZE_DATETIME }}. Your SIG has not submitted any enhancements for the {{ CYCLE }} cycle. If your SIG still plans to submit an enhancement, follow the [instructions here](https://groups.google.com/a/kubernetes.io/g/dev/c/slwAhtVqjXk/m/zz-eeXdFBgAJ) so the enhancements team can begin tracking. If your SIG is not planning to submit an enhancement, please reply to this message to confirm. Please reach out to [#release-enhancements](https://kubernetes.slack.com/archives/C02BY55KV7E) if you have any questions.
 
+#### Collection Monitoring and Triage (Weeks 1-4)
+
+- Monitor the Enhancements Tracking Board for new additions as SIGs opt-in KEPs for the release and assign an enhancement contact to each Issue.
+- For opted in Issues periodically (at least once a week):
+  - Ensure Issue is in the current milestone
+  - Ensure Issue has correct labels (has `tracked/yes`, does not have `tracked/no`, `stage/xxx` is accurate, ...)
+  - Ensure Fields in the Enhancement Tracking Board are populated and accurate (Status, Stage, SIG, ...)
+  - Evaluate if the enhancement satisfies all of the requirements for inclusion in the current release.
+    - Comment on the Issue with a status updating using one of the [templates from below](#enhancement-freeze-templates)
+    - Update `Enhancement Status` field for this Enhancement in the Enhancement Tracking Board (`Tracked` or `At Risk`)
+- Reach out to each [SIG on Slack](https://github.com/kubernetes/community/blob/master/sig-list.md) tagging one or more chairs or technical leads to communicate the Enhancements statuses. Example [communication templates](#sig-outreach-templates) can be found below.
+- Start syncing with Communications Team on giving an induction what's coming up for the release.
 - Send an email to [Kubernetes-Dev](https://groups.google.com/a/kubernetes.io/g/dev) that Enhancement freeze is coming and share current Enhancements status. Examples [1](https://groups.google.com/g/kubernetes-dev/c/-nTNtBBHL2Y/m/WfNzb_E1EAAJ).
+- Provide updates during release team meetings
+  - Use the `KEPs by Stage` insight from the Enhancement Tracking Board ([example](https://github.com/orgs/kubernetes/projects/98/insights/3))
 
-#### Enhancements Freeze (Week 3)
+#### Enhancements Freeze (Week 5)
```

Comment:
the weeks can differ from release to release slightly.. but we can keep it you think its not confusing

Reply:
Should I remove the number here?
If I do that then I can update the above section to `Collection Monitoring and Triage (Weeks 1-Enhancements Freeze)`

Reply:
Maybe rephrase headers to : "before Enhancement Freeze", "On Enhancements Freeze Day", "During the week after Enhancements Freeze" and so on... so define time spans. (you can also stick to the current form if it works..)

Reply:
Updated - PTAL